ocaml/idl/api_errors.ml

```diff
@@ -292,6 +293,7 @@ let restore_incompatible_version = "RESTORE_INCOMPATIBLE_VERSION"
 let restore_target_missing_device = "RESTORE_TARGET_MISSING_DEVICE"
 let restore_target_mgmt_if_not_in_backup = "RESTORE_TARGET_MGMT_IF_NOT_IN_BACKUP"
 
+let pool_management_cannot_be_set_on_system_network = "POOL_MANAGEMENT_CANNOT_BE_SET_ON_SYSTEM_NETWORK"
```

Comment:
Replacing `be_set_on` with `use` would save six characters and would be slightly clearer.